src/pcrlock/pcrlock.c

```diff
@@ -4724,6 +4724,16 @@ static int verb_make_policy(int argc, char *argv[], void *userdata) {
         return make_policy(arg_force, arg_recovery_pin);
 }
 
+static int verb_can_make_policy(int argc, char *argv[], void *userdata) {
+        _cleanup_(tpm2_context_unrefp) Tpm2Context *tc = NULL;
+        int r = tpm2_context_new(NULL, &tc);
```

Comment:
I think we should run tpm2_support() first, before trying to connect to the TPM. it checks a couple of other things, such as firmware support. Which kinda matters here, since we want a fully populated firmware log and such. After all TPMs might be avaulable that do not carry firmware measurements which would make them pretty useless for us. Hence let's do a thorough check first. We can always relax the rules should we feel the need to after all, and it's good to start with strict checks.